packages/opds-web-client/src/components/Book.tsx

```diff
@@ -83,6 +90,11 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
               { this.fields().map(field =>
                 field.value ? <div className={field.name.toLowerCase().replace(" ", "-")} key={field.name}>{field.name}: {field.value}</div> : null
               ) }
+              {
+                this.getMedium(this.props.book) && (
+                  <span>Media: {this.getMediumSVG(this.getMedium(this.props.book))}</span>
```

Comment:
I'm not sure it needs the "Media: " prefix, I think the SVG and text is enough here.

Reply:
I thought the same but I added it to be consistent with the previous information layout.